src/Index.cs

```diff
@@ -148,6 +148,7 @@ public Task<IndexStats> DescribeStats(MetadataMap? filter = null, CancellationTo
         string? indexNamespace = null,
         CancellationToken ct = default)
     {
+#if !NETSTANDARD2_0
```

Comment:
throw instead, rather than silently doing the operation in one go? (same for Fetch)

Reply:
I think it's better not to expose this overload on NS2.0 at all, and if-def the non-batched overload to just call the transport directly on NS2.0, as it was before recent commits - it's a single line anyway :)